README.md

```diff
@@ -58,9 +58,12 @@ There are a number of gems that need configuration to work:
 If you look in `docker-compose.yml` you'll notice that we've spun up a number of containers, not all of which may be useful for your project.
 
 1. **web**: Your core container which runs the `rails` server process.
-2. **worker**: A duplicate of your core app, but running `sidekiq` instead.
-3. **redis**: A `redis` instance for `sidekiq`.
-4. **localstack**: [`localstack`](https://github.com/localstack/localstack) is a suite of fake AWS services.
+2. **worker**: A duplicate of your core app, but running `sidekiq` instead. [Disabled by Default]
+3. **redis**: A `redis` instance for `sidekiq`. [Disabled by Default]
+4. **localstack**: [`localstack`](https://github.com/localstack/localstack) is a suite of fake AWS services. [Disabled by Default]
 5. **postgres**: Your application's database.
```

Comment:
Didn't this get switched to MySql? Should it say 'disabled by default'? Is there a `mysql` container?